integration/e2e/e2e_test.go

```diff
@@ -213,7 +220,13 @@ var _ = Describe("EndToEnd", func() {
 		BeforeEach(func() {
 			network = nwo.New(nwo.BasicKafka(), testDir, client, StartPort(), components)
 			network.MetricsProvider = "prometheus"
+
 			network.ChannelParticipationEnabled = true
+			fileRepoDir := filepath.Join(testDir, "filerepo")
+			network.ChannelParticipationFileRepoDir = fileRepoDir
```

Comment:
Why is there a single file repository _for the network_? It does not seem that this is a network-wide location.

Reply:
Done. Fixed by moving the file repo directory under the configured file ledger location, which is different per orderer node.